src/vs/editor/contrib/codeAction/browser/media/action.css

```diff
@@ -30,8 +30,8 @@
 	z-index: 5; /* make sure we are on top of the tree items */
 	content: "";
 	pointer-events: none; /* enable click through */
-	outline: 0px solid; /* we still need to handle the empty tree or no focus item case */
-	outline-width: 0px;
+	outline: 0px solid !important; /* we still need to handle the empty tree or no focus item case */
+	outline-width: 0px !important;
```

Comment:
Added back important tags - no issues in OSS with the hover/click, but in Insiders, the focus border will show up when clicking on a disabled code action.